.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+## Bug Report
+<!--
+    If you are not submitting a bug report, ignore this template and see
+      http://Chapel-Issues-Best-Practices.com  (TODO)
+  -->
+### Summary of Problem
+
+
+### Steps to Reproduce
+
+**Behavior:**
+<!-- What behavior did you observe when encountering this issue?  -->
+
+**Source Code:**
+<!-- Please provide source code that will reproduce the problem as an
+       attachment, an inline code block, or via a URL.  To the extent possible,
+       providing reduced reproducers of the problem will be appreciated. -->
+
+```chapel
+// You can insert your code inline if it's not too long
+// Otherwise, you can attach it as a ZIP file or provide a URL to it
```

Comment:
Do attachments need to be ZIPs?  If it were just one file, I'd just as soon not have to unzip it...  If they have multiple files, I'd probably have them figure out they need to tar and/or zip it.  I guess I'm saying I might just say "attach it or..." and drop "as a ZIP file" if GitHub issues doesn't require zips...